### PR TITLE
[15.0][MIG] sale_isolated_quotation

### PR DESCRIPTION
**Pull request by Freni-OSI**
Dependency module:
- [ ] #1945

### CODE REVIEW

File: sale_isolated_quotation/tests/test_sale_isolated_quotation.py [hunk 1]
```diff
@@ -52,13 +52,13 @@ def test_uninstall_hook(self):
             action = self.env.ref(action_id)
             ctx = ast.literal_eval(action.context)
             dom = ast.literal_eval(action.domain or "{}")
-            self.assertTrue("order_sequence" not in ctx)
-            self.assertTrue("default_order_sequence" not in ctx)
-            self.assertTrue("order_sequence" not in map(lambda l: l[0], dom))
+            self.assertFalse("order_sequence" not in ctx)
```
Comment (rousseldenis) [inline sale_isolated_quotation/tests/test_sale_isolated_quotation.py]: Why changing the assertion ? Strange

**Comment by rousseldenis** [comment]
@Freni-OSI Could you review #1945 ?

File: sale_isolated_quotation/tests/test_sale_isolated_quotation.py (hunk 1 above)
Comment (Freni-OSI) [reply sale_isolated_quotation/tests/test_sale_isolated_quotation.py]: @rousseldenis Improved. Thansk

**Comment by Crazor** [comment]
Any updates on this? Would like to use this on 15.0

**Pull request closed without merging**